github/operations.py

```diff
@@ -160,15 +167,15 @@ def create_update_file_contents(config, params, *args, **kwargs):
     params.update({'content': content})
     payload = {k: v for k, v in params.items() if
                v is not None and v != '' and v != {} and v != [] and k not in ['path', 'owner', 'name', 'org']}
-    if params.get('repo_type') == 'Organization':
-        endpoint = 'repos/{0}/{1}/contents/{2}'.format(params.get('org'), params.get('name'), params.get('path'))
-    else:
-        endpoint = 'repos/{0}/{1}/contents/{2}'.format(params.get('owner'), params.get('name'), params.get('path'))
+    endpoint = 'repos/{0}/{1}/contents/{2}'.format(
+        params.get('org') if params.get('repo_type') == 'Organization' else params.get('owner'), params.get('name'),
+        params.get('path'))
     return github.make_request(method='PUT', data=json.dumps(payload), endpoint=endpoint)
```

Comment:
the make_request signature would be:
make_request(method='PUT', data, endpoint, name, repository=None, owner=None):
        if repository:
             endpoint = "repos/" + repository
....
             

and the caller would be github.make_request(method='PUT', data=json.dumps(payload), name, endpoint=endpoint, repository=params.get("repository"), organization=params.get("organization"))

Reply:
Done